README.md

```diff
@@ -28,6 +34,8 @@ This will run every test in the default configuration.
 
 - [glow](https://repology.org/project/glow/packages) for rendering markdown in the terminal ([bat](https://repology.org/project/bat-cat/packages), [pygmentize](https://repology.org/project/pygments/versions) or [highlight](https://repology.org/project/highlight/packages) can be used as fallback, but glow's output is clearer and therefore glow is recommended)
 
+All dependencies are automatically installed if `xdg-ninja` is installed with Homebrew.
+
```

Comment:
I don't think this is necessary since package managers usually install dependencies, and I don't want to have to add this for every package manager that might be added in the future.

```suggestion
```